rcl/test/rcl/test_graph.cpp

```diff
@@ -852,6 +852,67 @@ check_graph_state(
   }
 }
 
+void
+check_entity_number(
```

Comment:
Nit: Consider this renaming? Just to be very explicit about the point of the check.
```suggestion
check_entity_count(
```

Reply:
👍  thanks! i will change it as you suggested.